src/services/category.js

```
@@ -19,6 +19,12 @@ const categoryService = {
   createCategory: async (data) => {
     const { name, appearance } = data
 
+    let category = await Category.findOne({ name })
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)